UnityProject/Assets/Scripts/Clothing/BackPack/InteractableStorage.cs

```diff
@@ -167,6 +178,12 @@ public bool WillInteract(HandApply interaction, NetworkSide side)
 		if (allowedToInteract == false) return false;
 		// Use default interaction checks
 		if (DefaultWillInteract.Default(interaction, side) == false) return false;
+		if (itemStorage.GetItemSlots().All(slot => slot.IsOccupied) && interaction.HandObject != null)
```

Comment:
since it is the same code as in the other WillInteract, extract this check into a private method